README.md

```diff
@@ -210,6 +211,26 @@ For dex.decred.org, reach out
 [on Element](https://matrix.to/#/!mlRZqBtfWHrcmgdTWB:decred.org?via=decred.org&via=matrix.org&via=planetdecred.org)
 to appeal.
 
+## Fees
+
+The dex does not charge fees, but users pay on-chain transaction fees.
+
+To ensure that on-chain transaction fees do not eat a significant portion of the
+order quantity, orders must be specified in increments of a minimum lot size.
+For example, if on-chain transaction fees worked out to $5, and a user was able
+to place an order to trade $10, they would lose half of their trade to
+transaction fees. So for high-fee assets, the operator must set a large lot
+size.
```

Comment:
So -> This is why

But...

Consider sticking to example mode here. Something like

> For chains with single-match fees of $5, if the operator wanted to limit possible fees to under 1% of the trade, the minimum lot size would need to be set to about $500.

Reply:
Definitely nicer.